std/Any.hx

```diff
@@ -31,6 +31,7 @@
 	to work with the actual value, it needs to be explicitly promoted
 	to another type.
 **/
+@:forwardVariance
```

Comment:
Should `Dynamic` be annotated with this too?

Reply:
This meta is for an abstract who wish to unify in type params and nested fields with an underlying type, but do not have, for one reason or another, a direct implicit cast to the underlying.

Why `Dynamic` does not need it:
- It is an abstract in name only, it has its own type - `TDynamic`.
- It does not have underlying type.
- It is already very unifiable in that regard.